doc/source/whatsnew/v1.4.0.rst

```diff
@@ -355,7 +355,7 @@ Groupby/resample/rolling
 - Bug in :meth:`pandas.DataFrame.ewm`, where non-float64 dtypes were silently failing (:issue:`42452`)
 - Bug in :meth:`pandas.DataFrame.rolling` operation along rows (``axis=1``) incorrectly omits columns containing ``float16`` and ``float32`` (:issue:`41779`)
 - Bug in :meth:`Resampler.aggregate` did not allow the use of Named Aggregation (:issue:`32803`)
--
+- Bug in :meth:`pandas.DataFrame.groupby.rolling` and :class:`pandas.api.indexers.FixedForwardWindowIndexer` leading to segfaults and window endpoints being mixed across groups (:issue:`43267`)
```

Comment:
Could you move this to the bug fix section in 1.3.3

Reply:
planning the 1.3.3 release tomorrow. changing milestone to 1.3.4. (1.3.4 release notes won't be added to master until after 1.3.3 is released)

Reply:
Moved to 1.3.4 per request